topostats/run_topostats.py

```diff
@@ -331,11 +331,12 @@ def process_scan(
                     results["Image Name"] = filtered_image.filename
                     results["Basename"] = image_path.parent
 
-            except Exception:
+            except Exception as e:
                 # If no results we need a dummy dataframe to return.
                 LOGGER.info(
                     f"[{filtered_image.filename}] : Errors occurred attempting to calculate grain statistics and DNA tracing statistics."
                 )
+                raise e
```

Comment:
Ah, thank you I put this in to debug why things weren't working and forgot to remove it.

Reply:
As an addendum I'm keen to refactor `Grains`/`GrainStats`/`DNATracing` and get much more isolated functions in place with lots of `try:...except:` in place and associated tests to capture errors rather than the vastly broad `try:... except:` in place as it really helps pin-point where errors arise.